app/Providers/ModelServiceProvider.php

```diff
@@ -52,6 +52,10 @@ public function boot()
 
         // When Reviews are saved create an event for them.
         Review::saved(function ($review) {
+            info('review', ['campaign' => $review->post->campaign_id]);
+            // Update the "counter cache" on this campaign:
+            $review->post->campaign->refreshCounts();
```

Comment:
Oh wait I see that this count is getting updated whenever there is a review, and the job is just to make sure things don't get wildly out of sync. Is that right?

Reply:
Yes! The job is to "re-sync" if things somehow got wonky, and to take into account newly submitted posts (which don't trigger the on-demand count).